rpc/src/test/scala/server/ServerConfigTests.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc
+package server
+
+import cats.Id
+import cats.data.Kleisli
+import cats.implicits._
+import freestyle._
+import freestyle.implicits._
+import freestyle.config.implicits._
+import io.grpc.Server
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, Future}
+
+class ServerConfigTests extends RpcTestSuite {
+
+  import implicits._
+
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+
+  "ServerConfig.Op" should {
+
+    "load the port specified in the config file" in {
+
+      val serverF = BuildServerFromConfig[ServerConfig.Op]("rpc.server.port").interpret[Future]
+
+      serverF map (_.getPort shouldBe port)
```

Comment:
Because this is a `Future`, this test is not verifying the value. I've tried different values for `port` and it succeeds for all of them